pyso/stack_search_module.py

```diff
@@ -33,18 +33,19 @@ def __init__(self, access_token, key='uKDJS)Ck32rrSYrweVOoOQ(('):
             answer_id = question.get("items", [{}])[0].get('accepted_answer_id', -1)
 
             # Grab the answer's JSON
-            answer = None
-            async with aiohttp.ClientSession() as session_thing:
-                answer = await fetchAnswer(session_thing, answer_id)
+            answer = await fetchAnswer(session, answer_id)
             
+            if answer.get('items', None) == None:
```

Comment:
[Use `is None` instead of `== None`](https://stackoverflow.com/questions/14247373/python-none-comparison-should-i-use-is-or)